### PR TITLE
update read me

### PR DESCRIPTION
**Pull request by edewit**
more read me updates

### CODE REVIEW

**Comment by matzew** [comment]
Any comment, @sebastienblanc ?

**Comment by abstractj** [comment]
@danielpassos I would like your help reviewing it. I'm planning to test this README today and merge after your plus one.

File: Shoot/www/js/app.js [hunk 1]
```diff
@@ -14,7 +14,8 @@ angular.module('starter', ['ionic', 'starter.controllers', 'starter.services'])
     oauth2.addGoogle({
       name: 'gplus',
       settings: {
-        clientId: '<your clientId e.g. 517285908032-nnkcrot1727fmd738pug6clbqlgosffs.apps.googleusercontent.com>',
+        //replace with your own if you want to...
+        clientId: '241956090675-gkeh47arq23mdise57kf3abecte7i5km.apps.googleusercontent.com',
```
Comment (danielpassos) [inline Shoot/www/js/app.js]: I think it's your client ID :D

Comment (edewit) [inline Shoot/www/js/app.js]: that is the line that is removed ;)

File: Shoot/README.md [hunk 2]
```diff
@@ -15,4 +15,78 @@ $ cordova prepare ios
...
+- enter password
+- accept policy
+- send confirmation code to SMS
+- once received enter code
+
+### Step2: Create a new app on facebook console
+
+- Click apps-> Create a new app
+- add display name: Shoot
+- deal with difficult catcha
```
Comment (danielpassos) [inline Shoot/README.md]: I need to choice the platform

screenshot 2015-02-11 15 31 32

File: Shoot/README.md [hunk 3]
```diff
@@ -15,4 +15,78 @@ $ cordova prepare ios
...
+- configure Advanced setup:
+	- Native or desktop app? NO
+	- Client OAuth Login YES
+	- Embedded browser OAuth Login YES
+    
+### Step3: Configure Shoot app cordova client    
+
+Open app.js and update `clientId` and `clientSecret` with you `App ID` and `App Secret` and in case of iOS change `fbYYY` to `fb<App ID>` to the <project-name>-Info.plist:
```
Comment (danielpassos) [inline Shoot/README.md]: Sorry, it's probably obvious to the Cordova guys, but I have no idea where the app.js file is

File: Shoot/README.md (hunk 2 above)
Comment (edewit) [inline Shoot/README.md]: that wasn't there before, no clue

File: Shoot/README.md [hunk 4]
```diff
@@ -15,4 +15,78 @@ $ cordova prepare ios
...
+- accept policy
+- send confirmation code to SMS
+- once received enter code
+
+### Step2: Create a new app on facebook console
+
+- Click apps-> Create a new app
+- add display name: Shoot
+- deal with difficult catcha
+- configure Advanced setup:
```
Comment (danielpassos) [inline Shoot/README.md]: Settings -> Advenced

**Comment by danielpassos** [comment]
I'm testing Facebook on iOS and geeting

```
Check dependencies

Swift is unavailable on iOS earlier than 7.0; please set IPHONEOS_DEPLOYMENT_TARGET to 7.0 or later (currently it is '6.0').
warning: no rule to process file '/Users/passos/Development/Projects/AeroGear/aerogear-cordova-cookbook/Shoot/platforms/ios/Shoot/Plugins/org.jboss.aerogear.cordova.oauth2/AeroGearHttp.h' of type sourcecode.c.h for architecture x86_64
warning: no rule to process file '/Users/passos/Development/Projects/AeroGear/aerogear-cordova-cookbook/Shoot/platforms/ios/Shoot/Plugins/org.jboss.aerogear.cordova.oauth2/Bridging-Header.h' of type sourcecode.c.h for architecture x86_64
warning: no rule to process file '/Users/passos/Development/Projects/AeroGear/aerogear-cordova-cookbook/Shoot/platforms/ios/Shoot/Plugins/org.jboss.aerogear.cordova.oauth2/AppDelegate+url.h' of type sourcecode.c.h for architecture x86_64
```

Any idea?

**Comment by danielpassos** [comment]
For Android... After I run

```
cordova platform add ios android
```

I got the instruction:

```
open platforms/android/build.gradle and under the `android` section add:
```

But I can not find that file

**Comment by edewit** [comment]
for iOS you'll need to open xcode project file into xcode and make all the changes to the downloaded project as stated in the [readme linked in this document](https://github.com/aerogear/aerogear-oauth2-cordova/blob/master/README.md#workaround-for-ios).

**Comment by edewit** [comment]
for android what version of the android platform have you installed with cordova? You can find out by running `cordova platform ls` output should be 
```
~/w/d/a/Shoot git:readme ❯❯❯ cordova platform ls
Installed platforms: android 3.6.4
```

**Comment by danielpassos** [comment]
@abstractj @matzew @edewit 

In the last 2 days I was playing a bit with Cordova Shoot and Share app to test this doc.

## README

I did some improve on it and I know it still needs some change for iOS in GDrive, feel free to add/improve it. Here is my branch => https://github.com/danielpassos/aerogear-cordova-cookbook/tree/readme

## App

Unfortunately I could not run that because

### iOS 

Seems the library used is outdate and I got this problem when I try run.

ios-problem

### Android

The application does not have an default activity to run the application

android-activity

Not sure if I did something wrong, so feel free to ping me to test it again.

**Comment by danielpassos** [comment]
Sent a PR about README improvements https://github.com/edewit/aerogear-cordova-cookbook/pull/1

**Comment by edewit** [comment]
The MultiPartData is no longer part of the plugin could you try and remove the plugin and adding it from with the github url:
```bash
cordova plugin rm org.jboss.aerogear.cordova.oauth2
cordova plugin add https://github.com/aerogear/aerogear-oauth2-cordova
```

**Comment by edewit** [comment]
@danielpassos merged in your branch could you check?

**Comment by abstractj** [comment]
@edewit did you rebase against master? I found several conflicts, but is safer if you solve the conflicts than me.

**Comment by edewit** [comment]
right, I'll rebase

**Comment by abstractj** [comment]
@edewit landed!

**Pull request merged**